src/vss_tools/exporters/binary.py

```diff
@@ -126,12 +129,15 @@ def cli(
     overlays: tuple[Path],
     quantities: tuple[Path],
     units: tuple[Path],
+    types: tuple[Path],
+    types_output: Path,
+    extend_all_attributes: bool,
```

Comment:
Not using that, right?

Reply:
extend_all_attributes is probably not needed. The other two are referenced at other places in the code so I assume they are needed.

Reply:
Yeah so please get rid of the unused ones